docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
+  - Click **Save**.
+- Click the **Save** button and wait for Strapi to restart.
+
+## 6. Add content to "Restaurant" Content Type
 
 Navigate to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/restaurant?source=content-manager).
 
-- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Name** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
+- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Restaurant** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
 - Click **Save**.
 
 You will see your restaurant listed in the entries.
 
-## 6. Set roles and permissions
+## 7. Add categories to the "Category" Content Type
+
+Navigate to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/category?source=content-manager).
+
+- Click on **+ Add New Category** button. Type `Convenient` in the **Category** field. Select `Strapi Restaurant`, on the right, from **Restaurant (0)**
+- Click **Save**.
+
+You will see the **Convenient** category listed in the entries.
+
+- Click on **+ Add New Category** button. Type `Time Saving` in the **Category** field. **DO NOT ADD IT HERE** to `Strapi Restaurant`.
+- Click **Save**.
+
+You will see the **Time Saving** category listed in the entries.
+
+Navigate back to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/restaurant?source=content-manager).
+
+- Click on `Strapi Restaurant`
+- On the right, under **Categories(1)**, `select` the `Add an item...`, and add **Time Saving** as a category for this restaurant.
+
+**Two different ways** to use the **relation** field type to add and connect relations to other fields.
+
+## 8. Set roles and permissions
 
-Navigate to [**PLUGINS** - **Roles & Permission**](http://localhost:1337/admin/plugins/users-permissions/roles).
+Navigate to [**PLUGINS** - **Roles & Permissions**](http://localhost:1337/admin/plugins/users-permissions/roles).
 
-- Click the **pencil** edit icon to the right of the **Public** Role.
-- Scroll down under **Permissions**, find **Restaurant**. Click the checkbox next to **find**.
+- Click the **Public** Role.
+- Scroll down under **Permissions**, find **Restaurant**. Click the checkbox next to **find** and **findone**.
+- Repeat and scroll down under **Permissions**, find **Category**. Click the checkbox next to **find** and **findone**.
 - Click **Save**.
 
-## 7. Consume the Content Type API
+## 9. Consume the Content Type API
```

Comment:
Consume the Content Type's API

Reply:
Thank you for clearing this up.  I should have asked.  Corrected in both Alpha and Beta.